temoa/data_io/hybrid_loader.py

```
@@ -552,24 +568,32 @@ def _load_time_season_sequential(
         if filtered_data:
             ordered_data = [row[0:3] for row in filtered_data]
             self._load_component_data(data, M.ordered_season_sequential, ordered_data)
-            seq_data = sorted(list(set((row[1],) for row in filtered_data)))
+            seq_data = sorted(list({(row[1],) for row in filtered_data}))
```

Ruff (0.14.0)

Unnecessary `list()` call within `sorted()`

Remove the inner `list()` call

(C414)